detect_secrets/core/baseline.py

```diff
@@ -256,3 +258,15 @@ def _get_git_tracked_files(rootdir='.'):
         return set(git_files.decode('utf-8').split())
     except subprocess.CalledProcessError:
         return None
+
+
+def _get_files_recursively(rootdir):
+    """Sometimes, we want to use this tool with non-git repositories.
+    This function allows us to do so.
+    """
+    output = []
+    for root, dirs, files in os.walk(rootdir):
```

Comment:
Super nit: you can [use _ for dirs](https://github.com/python-security/pyt/blob/master/pyt/__main__.py#L39)